index.js

```diff
@@ -158,7 +160,9 @@ export function execa(file, args, options) {
 
 	spawned.all = makeAllStream(spawned, parsed.options);
 
-	return mergePromise(spawned, handlePromiseOnce);
+	addPipeMethods(spawned);
+	mergePromise(spawned, handlePromiseOnce);
```

Comment:
`addPipeMethods()` relies on `spawned` being directly mutated, so we make it clear here.